README.md

```diff
@@ -1,9 +1,9 @@
 # Getting Started with a multi-node Kubernetes Provider
 
-Download cluster-up repo
+Download this repo
 ```
-git clone https://github.com/davidvossel/cluster-up.git
-cd cluster-up
+git clone https://github.com/kubevirt/kubevirtci.git
+cd kubevirtci/cluster-up
```

Comment:
I think this should just be ```cd kubevirtci```

All the scripts are meant to be acted on in the top level kubevirtci dir (where the make target lives), not within cluster-up directly. The scripts do use absolute paths though, so it's likely that they'll work fine regardless of what directory they are executed in. 

However ```make clusterup``` only works in kubevirtci, not kubevirtci/cluster-up